packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -190,16 +190,24 @@ export default class Dropdown extends React.Component {
       [`${prefix}--label--disabled`]: disabled,
     });
 
+    const helperId = !helperText
+      ? undefined
+      : `dropdown-helper-text-${this.dropdownInstanceId}`;
+    const labelId = `dropdown-label-${this.dropdownInstanceId}`;
+    const fieldLabelId = `dropdown-field-label-${this.dropdownInstanceId}`;
+
     const title = titleText ? (
-      <label htmlFor={id} className={titleClasses}>
+      <span id={id} className={titleClasses}>
```

Comment:
It seems like the `id` being used here and in `ListBox` is causing a DAP violation: 
<img width="530" alt="Screen Shot 2020-01-07 at 4 41 22 PM" src="https://user-images.githubusercontent.com/17281178/71935504-8ffa5c80-316c-11ea-9c0a-bdd67022ee73.png">

Reply:
Great catch @abbeyhrt! Was an error in rebasing. Fixed.